core/src/test/java/io/grpc/ClientInterceptorsTest.java

```diff
@@ -351,6 +352,33 @@ protected void checkedStart(ClientCall.Listener<RespT> responseListener, Metadat
     ArgumentCaptor<Status> captor = ArgumentCaptor.forClass(Status.class);
     verify(listener).onClose(captor.capture(), any(Metadata.class));
     assertSame(error, captor.getValue().getCause());
+
+    // Make sure nothing bad happens after the exception.
+    ClientCall<?, ?> noop = ((CheckedForwardingClientCall<?, ?>)interceptedCall).delegate();
```

Comment:
In CheckedForwardingClientCall.start, if an exception occurs, the delegate becomes the noop call.  NOOP_CALL has a bunch of noop methods that show up as uncovered, which this test now covers.  This test makes sure they are actually noops, by passing in bad data and making sure they really don't do anything.

The code seems arbitrary to me, but at least now we have tests proving the methods are doing the right thing.